### ISSUE TITLE
Geocoder not working

### ISSUE THREAD

**Issue by bonan999** [AHAAAAAAA/PokemonGo-Map#1504]
## Current Behavior
When trying to start server the error message ```
The given key has gone over the requests limit in the 24'
geopy.exc.GeocoderQuotaExceeded: The given key has gone over the requests limit in the 24 hour period or has submitted too many requests in too short a period of time.
``` shows up. But if I check the quota for this API i haven't made any requests. Different API key for the geocoder API?

## Context
Unable to start server

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Version used: latest master build
* Environment name and version (e.g. Python 2.7): 2.7
* Operating System and version (desktop or mobile): Mac OSX 10.11.5

**Comment by MRokas** [comment]
> the given key has gone over the requests limit in the 24 hour period or has submitted too many requests in too short a period of time.

Here's your problem. Check twice if you're using right key.

**Comment by bonan999** [comment]
Yes, I know! The weird thing is that I've been able to use this key before with the exact same command.

**Comment by MRokas** [comment]
> before

Well, you cant use it anymore, you went API overlimit. Just get new key.

**Comment by bonan999** [comment]
I guess I'll have to do that, just weird that no usage showed up on google. That's why I started to think that the key that I generated wasn't being used for geocoding

**Comment by bonan999** [comment]
Still the same error message after generating a new key

**Comment by shermsaludez** [comment]
Have you found the fix? I have the same problem
